app/js/disease-associations-controller.js

```diff
@@ -68,34 +68,13 @@ angular.module('cttvControllers')
 
         // facets changed?
         if (!_.isEqual(new_state[facetsId], old_state[facetsId]) || !new_state[facetsId]) {
-            getFacets(new_state[facetsId]);
+            $scope.getFacets(new_state[facetsId]);
         }
 
     };
 
-    var initFilterByFile =function(){
-        $scope.fileName = "";
+    $scope.targetArray = [];
```

Comment:
SO this targetArray variable is the one that gets populated with all the target Ids and disease-association-table-directive watches is and triggers association lookup and table changes. So i had to keep it here, which is not too bad, but what is bad is that later on i update it from my new filter-by-file-targets-directive.js